### ISSUE TITLE
prompt those who haven't previously confirmed TOS agreement to agree

### ISSUE THREAD

**Issue by sneak** [steemit/condenser#2560]
As a user who signed up via anonsteem or elsewhere, it is important to steemit that I be offered a chance to agree to the current steemit TOS when attempting to use my account on steemit.com, and that that agreement be recorded in the conveyor tagging database.

**Comment by r351574nc3** [comment]
What about accounts created through steemit/steemconnect? These accounts don't have users to accept the TOS. Is that a separate issue for steemit/steemconnect to handle at account creation time?

**Comment by sneak** [comment]
Accounts created via steemconnect are covered by this under "elsewhere".  Accounts created by steemit are covered under the following:

https://github.com/steemit/faucet/issues/232  (future ones created by steemit via signup app `faucet`)

https://github.com/steemit/condenser/issues/2559  (past users created by steemit via `condenser`)

**Comment by bnchdrff** [comment]
blocked by steemit/conveyor#26

**Comment by pkattera** [comment]
Trigger the following lightbox for any user who has previously not accepted Steemit's terms of service and privacy policy on their next visit to steemit.com. This includes existing Steemit users who have not accepted the policies, or users from AnonSteem / Blocktrades.

![screen shot 2018-03-18 at 10 49 06 pm](https://user-images.githubusercontent.com/1121139/37580001-9e90bef4-2afe-11e8-96a2-3b3ea11361bf.png)

Text:

> **Please review our terms to continue**
> 
> Hi [@username]
> 
> Our Terms of Service set the rules and guidelines that you must accept and follow in order to use Steemit and our Privacy Policy explains the type of information we collect and what we do with it. Don't worry, we don't sell your data to anyone. We do use it to meet legal requirements and to learn how to make Steemit better.
> 
> I agree to Steemit's Terms of Service
> 
> I agree to Steemit's Privacy Policy

**Comment by gl2748** [comment]
Relates to:
https://github.com/steemit/faucet/issues/232

**Comment by pkattera** [comment]
Links to:
- Terms of Service: https://steemit.com/tos.html
- Privacy Policy: https://steemit.com/privacy.html

**Comment by pkattera** [comment]
Ideally, this issue will be implemented along with steemit/faucet#232

**Comment by gl2748** [comment]
Share the tos copy between both.

**Comment by bnchdrff** [comment]
i didn't implement as shown in the mockups above; revised PR incoming
